npm/test-system.js

```diff
@@ -77,7 +78,7 @@ module.exports = function (exit) {
                 return next();
             });
         }
-    ], exit);
+    ], done);
```

Comment:
The scripts are now less confusing, with unique names for the context-specific callback.